truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -356,17 +365,23 @@ def copy_into_build_dir(from_path: Path, path_in_build_dir: str):
 
             config.runtime.predict_concurrency = TRTLLM_PREDICT_CONCURRENCY
 
-            config.base_image = BaseImage(
-                image=BRITON_TRTLLM_BASE_IMAGE if USE_BRITON else TRTLLM_BASE_IMAGE,
-                python_executable_path=TRTLLM_PYTHON_EXECUTABLE,
-            )
-            config.requirements.extend(
-                BRITON_BASE_TRTLLM_REQUIREMENTS
-                if USE_BRITON
-                else BASE_TRTLLM_REQUIREMENTS
-            )
+            if not is_audio_model:
+                config.base_image = BaseImage(
+                    image=BRITON_TRTLLM_BASE_IMAGE if USE_BRITON else TRTLLM_BASE_IMAGE,
+                    python_executable_path=TRTLLM_PYTHON_EXECUTABLE,
+                )
+            
+                config.requirements.extend(
+                    BRITON_BASE_TRTLLM_REQUIREMENTS
+                    if USE_BRITON
+                    else BASE_TRTLLM_REQUIREMENTS
+                )
 
-            config.model_metadata["tags"] = [OPENAI_COMPATIBLE_TAG]
+                config.model_metadata["tags"] = [OPENAI_COMPATIBLE_TAG]
+            else:
+                config.requirements.extend(AUDIO_MODEL_TRTLLM_REQUIREMENTS)
```

Comment:
What image ends up being used if no base image is specified / were you able to test the trtllm installation in it?

Reply:
We don't use a base image here. The underlying base image is just the GPU base image that comes with Truss. Yes, the trtllm installation was tested.